helmfile/values/userCRDs/userCRDs.yaml.gotmpl

```diff
@@ -1,8 +1,12 @@
 userCRDs:
     enabled: {{ .Values.gatekeeper.allowUserCRDs.enabled }}
     admin:
-        users: {{ .Values.clusterAdmin.users }}
-        groups: {{ .Values.clusterAdmin.groups }}
+        users:
+        - kubernetes-admin
```

Comment:
Wondering if this should be configured in the default config instead 🤔

Reply:
I like that idea :+1:

Reply:
@OlleLarsson @robinAwallace I am not sure how you want this, please have a look at [my latest commit](https://github.com/elastisys/compliantkubernetes-apps/pull/1872/commits/73708d62e892fd9c07581de05de0d960de6bd91b) if it is something like this you were thinking